src/energyplus/ForwardTranslator/ForwardTranslateChillerElectricASHRAE205.cpp

```diff
@@ -0,0 +1,218 @@
+          ambientTempOk = true;
+        }
+      }
+    } else if (openstudio::istringEqual("Zone", ambientTemperatureIndicator)) {
+      // Ambient Temperature Zone Name: Optional Object
+      if (boost::optional<ThermalZone> _ambientTemperatureZone = modelObject.ambientTemperatureZone()) {
+        if (boost::optional<IdfObject> _owo = translateAndMapModelObject(_ambientTemperatureZone.get())) {
+          idfObject.setString(Chiller_Electric_ASHRAE205Fields::AmbientTemperatureZoneName, _owo->nameString());
+          ambientTempOk = true;
+        }
+      }
+    } else if (openstudio::istringEqual("Outdoors", ambientTemperatureIndicator)) {
+      ambientTempOk = true;
+    }
+    if (!ambientTempOk) {
+      LOG(Warn, "For " << modelObject.briefDescription() << " the Ambient Temperature Indicator is " << ambientTemperatureIndicator
+                       << " but the required objects don't match. Falling back to 'Outdoors'");
+      ambientTemperatureIndicator = "Outdoors";
+    }
+
+    idfObject.setString(Chiller_Electric_ASHRAE205Fields::AmbientTemperatureIndicator, ambientTemperatureIndicator);
+
+    if (openstudio::istringEqual("Outdoors", ambientTemperatureIndicator)) {
+      std::string oaNodeName = modelObject.nameString() + " OA Node";
+      if (boost::optional<std::string> s_ = modelObject.ambientTemperatureOutdoorAirNodeName()) {
+        oaNodeName = s_.get();
+      }
+
+      idfObject.setString(Chiller_Electric_ASHRAE205Fields::AmbientTemperatureOutdoorAirNodeName, oaNodeName);
+
+      IdfObject _oaNodeList(openstudio::IddObjectType::OutdoorAir_NodeList);
+      _oaNodeList.setString(0, oaNodeName);
+      m_idfObjects.push_back(_oaNodeList);
+    }
+
+    // Chilled Water Node Names: Required Nodes
+    if (auto node_ = modelObject.chilledWaterInletNode()) {
+      idfObject.setString(Chiller_Electric_ASHRAE205Fields::ChilledWaterInletNodeName, node_->nameString());
+    }
+
+    if (auto node_ = modelObject.chilledWaterOutletNode()) {
+      idfObject.setString(Chiller_Electric_ASHRAE205Fields::ChilledWaterOutletNodeName, node_->nameString());
+    }
+
+    // Optional Nodes
+
+    // Condenser
+    if (auto node_ = modelObject.condenserInletNode()) {
+      idfObject.setString(Chiller_Electric_ASHRAE205Fields::CondenserInletNodeName, node_->nameString());
+    }
+
+    if (auto node_ = modelObject.condenserOutletNode()) {
+      idfObject.setString(Chiller_Electric_ASHRAE205Fields::CondenserOutletNodeName, node_->nameString());
+    }
+
+    // Heat Recovery
+    if (auto node_ = modelObject.heatRecoveryInletNode()) {
+      idfObject.setString(Chiller_Electric_ASHRAE205Fields::HeatRecoveryInletNodeName, node_->nameString());
+    }
+
+    if (auto node_ = modelObject.heatRecoveryOutletNode()) {
+      idfObject.setString(Chiller_Electric_ASHRAE205Fields::HeatRecoveryOutletNodeName, node_->nameString());
+    }
```

Comment:
I'm leaving this but it'll never be used for now (since the Heat Recovery connection is disabled at model time)